mgwr/gwr.py

```diff
@@ -285,7 +291,7 @@ def _local_fit(self, i):
             return influ, resid, predy, betas.reshape(-1), w, Si, tr_STS_i, CCT
 
     def fit(self, ini_params=None, tol=1.0e-5, max_iter=20, solve='iwls',
-            lite=False, pool=None):
```

Comment:
This is a hard-breaking change we should avoid. I suggest keeping the keyword and warning when it is not None.